src/app/collection/pages/collection-ncyte/header-ncyte/header-ncyte.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'clark-header-ncyte',
+  templateUrl: './header-ncyte.component.html',
+  styleUrls: ['./header-ncyte.component.scss']
+})
+export class HeaderNcyteComponent implements OnInit {
+
+  constructor() { }
+
```

Comment:
If we are using a browse button there should be a function used to call browse items I believe.